module/FileNVRAM.c

```diff
@@ -359,7 +388,18 @@ static void readplist()
                 
                 if (plistSize && read(fh, plistBase, plistSize) == plistSize)
                 {
-                    XMLParseFile( plistBase, &gPListData );
+                    // cleaning nvram.plist from \n and \t.
+                    // data tag have those bytes after the serialization:
+                    //
+                    //  <key>SystemAudioVolume</key>
+                    //  <data>
+                    //      fw==
+                    //  </data>
+                    //
+                    // but I want it to be: <data>fw==</data>
+                    //
+                    // so some garbage ends up in /chosen/nvram. just remove them!
+                    XMLParseFile(strdelchar(strdelchar(plistBase, '\n'), '\t'), &gPListData);
```

Comment:
Yes, I agree that the plist has \n and \t's in it, and I'm fine with the code here (for compat with older chameleon versions), I'm just wondering if it's a bug in chameleon's XMLParseFile function. And if so, then that should be fixed as well.